src/data.js

```diff
@@ -93,6 +93,27 @@ module.exports = [
       'Maker',
     ],
   },
+  {
+    name: 'Asur Bernardo',
+    description: "Back end developer with no aesthetic sense. Full stack with reservations. Open-source enthusiast. Continuous learner!',
```

Comment:
You probably want to do:
```suggestion
    description: 'Back end developer with no aesthetic sense. Full stack with reservations. Open-source enthusiast. Continuous learner!',
```

Reply:
Whooops! That was dumb, thanks for the catch! :wink: